.taskcluster.yml

```diff
@@ -123,99 +123,126 @@ tasks:
                   extra:
                     treeherder:
-                      symbol: D-PR
+                      symbol: D
                   metadata:
-                    name: 'Fenix - Decision task (Pull Request #${pull_request_number})'
-                    description: 'Building and testing the Fenix - triggered by [#${pull_request_number}](${pull_request_url})'
-        - $if: 'tasks_for == "github-push" && head_branch[:10] != "refs/tags/"'
-          then:
-            $mergeDeep:
-              - {$eval: 'default_task_definition'}
-              - scopes:
-                  - ${assume_scope_prefix}:branch:${short_head_branch}
-                payload:
-                  command:
-                    - >-
-                      git fetch ${repository} ${head_branch}
-                      && git config advice.detachedHead false
-                      && git checkout FETCH_HEAD
-                      && python automation/taskcluster/decision_task.py push
-                extra:
-                  treeherder:
-                    symbol: D
-                metadata:
-                  name: Fenix - Decision task
-                  description: Schedules the build and test tasks for Fenix.
-        - $if: 'tasks_for == "github-release" && event["action"] == "published"'
-          then:
-            $mergeDeep:
-              - {$eval: 'default_task_definition'}
-              - scopes:
-                - ${assume_scope_prefix}:release
-                payload:
-                  command:
-                    - >-
-                      git fetch ${repository} refs/tags/${head_rev}
-                      && git config advice.detachedHead false
-                      && git checkout FETCH_HEAD
-                      && python automation/taskcluster/decision_task.py beta ${event.release.tag_name}
-                extra:
-                  treeherder:
-                    symbol: beta-D
-                metadata:
-                  name: Fenix Beta Decision Task
-                  description: Building and releasing Fenix to the beta channel - triggered by release ${event.release.tag_name}
-        - $if: 'tasks_for == "cron"'
-          then:
-            $mergeDeep:
-              - {$eval: 'default_task_definition'}
-              - scopes:
-                  - $if: 'trust_level == 3'
-                    then: assume:hook-id:project-mobile/fenix-nightly
-                    else: assume:hook-id:project-mobile/fenix-nightly-staging
-                routes:
-                  $if: 'trust_level == 3'
-                  then:
-                    - notify.email.fenix-eng-notifications@mozilla.com.on-failed
-                payload:
-                  $let:
-                    staging_flag:
-                      $if: 'trust_level == 3'
-                      then: ''
-                      else: '--staging'
-                  in:
+                    name: Fenix - Decision task
```

Comment:
Tiny nit: I'm wonderinf whether we should amend this to say "master pushes" too or is this good enough?